docs/configuration/configuration.md

```diff
@@ -213,6 +213,12 @@ tls_config:
 
 # Optional proxy URL.
 [ proxy_url: <string> ]
+# Comma-separated string that can contain IPs, CIDR notation, domain names
```

Comment:
I do wonder whether this should reference a <proxy_config> somehow (even if that's a bit hard to do when it's not a nested thing, but I think it would be clearer.)

Reply:
Well in an ideal world we would reference a http_config .. but it's harder to read. .. we do this in alertmanager